great_expectations/data_context/store/html_site_store.py

```diff
@@ -290,9 +292,9 @@ def get_url_for_resource(self, resource_identifier=None, only_if_exists=True):
                 return store_backend.get_url_for_key(key)
 
     def _validate_key(self, key):
-        if not isinstance(key, SiteSectionIdentifier):
+        if not isinstance(key, (SiteSectionIdentifier, GeCloudIdentifier)):
             raise TypeError(
-                "key: {!r} must a SiteSectionIdentifier, not {!r}".format(
+                "key: {!r} must a SiteSectionIdentifier or RenderedSectionIdentifier, not {!r}".format(
```

Comment:
```suggestion
                "key: {!r} must a SiteSectionIdentifier or GeCloudIdentifier, not {!r}".format(
```

I don't think we'd want to implement a new `RenderedSectionIdentifier` since there's no notion of a data docs "section" in cloud.

Reply:
Updated